### ISSUE TITLE
Would be nice to post process the doc block for links 

### ISSUE THREAD

**Issue by chenglou** [rescript-lang/rescript-vscode#142]
Low pri.

For a doc block such as:

```res
@ocaml.doc("
  The top level provides some generic sort related utilities.
  It also has two specialized inner modules
  [`Belt.SortArray.Int`](...) and [`Belt.SortArray.String`](...)
")
```

it'd be nice to postprocess those links so that they're clickable. The IDE supports that. Caveat: doesn't support jumping to a line, so most of the links will be useless for now.

**Comment by zth** [comment]
It was a while since I dug into the specifics, but for the VSCode part I think it might be possible to link to a command from the extension. If so, one could imagining having a command that opens a file and scrolls to a certain position, but programmatically. I've done similar things before, so I'll have a look around if that's possible and report back here.

**Comment by zth** [comment]
Yes, this seems possible: https://code.visualstudio.com/api/extension-guides/command#command-uris
